samples/sdl_joystick_gamepad/Makefile

```diff
@@ -0,0 +1,10 @@
+DEBUG = y
```

Comment:
Please make commit titles consistent: one of the sample commit uses "sdl:" the other one uses "samples:" as prefix, although they seem to do almost exactly the same.